src/main/java/org/kafkahq/repositories/TopicRepository.java

```diff
@@ -99,11 +103,13 @@ public boolean isListViewMatch(TopicListView view, String value) {
 
     public Topic findByName(String clusterId, String name) throws ExecutionException, InterruptedException {
 
-        Optional<Topic> topics = Optional.empty();
+        Optional<Topic> topic = Optional.empty();
         if(isTopicMatchRegex(getTopicFilterRegex(),name)) {
-            topics = this.findByName(clusterId, Collections.singletonList(name)).stream().findFirst();
+            topic = this.findByName(clusterId, Collections.singletonList(name)).stream().findFirst();
+            topic.ifPresent(t -> t.getUsers().addAll(userRepository.findByResourceType(clusterId, ResourceType.TOPIC, t.getName())));
```

Comment:
Seems to be tricky here, you append data to a normally immutable object. 
I prefer to pass data on construct on `Topic` class instead